VERSION

```diff
@@ -0,0 +1 @@
+1.0.0-pre1
```

Comment:
Yeah, my intent is to have the VERSION file in this vector-datasource repo establish what the SemVer version is for the Tilezen API (the content), and that should be maintained before a git release is tagged (release wiki has been updated to this effect). The upside is it's easy to read the VERSION file directly in the repo without going thru a bunch of git hoops, and if the files become disassociated from a git checkout the VERSION is still carried along.

+1 for adding a lint / CI check to ensure that the version matches the git tag.

Agree that the Mapzen Vector Tiles service VERSION should be somewhere else, sounds like Chef in json. Should there be a new version endpoint to query this?